django_cloud_deploy/tool_requirements.py

```diff
@@ -157,14 +157,16 @@ def check(cls):
             subprocess.check_call(command, stdout=subprocess.DEVNULL,
                                   stderr=subprocess.DEVNULL)
         except subprocess.CalledProcessError:
-            # TODO: Handle for multiple OS's
-            # TODO: Check output for error message given when user has
-            # not ran the command
-            msg = ('Docker is installed but not correctly set up.'
-                   'Use the following command to fix it: \n'
-                   '$ sudo groupadd docker\n'
-                   '$ sudo usermod -a -G docker $USER\n'
-                   'Then log out/log back in')
+            if sys.platform.startswith('linux'):
+                msg = ('Docker is installed but not correctly set up.'
+                       'Use the following command to fix it: \n'
+                       '$ sudo groupadd docker\n'
+                       '$ sudo usermod -a -G docker $USER\n'
+                       'Then log out/log back in')
+            else:
+                msg = ('Docker is installed but was unable to run the command.'
+                       'Make sure docker is running, if it is open '
```

Comment:
Maybe "Make sure that docker is running and on the path. If you recently installed it, you may have to <whatever you need to do to ensure that the path gets updated>."